### PR TITLE
New QA handler

### PR DESCRIPTION
**Pull request by maxbartolo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ $ dynalab-cli test -n <name_of_your_model>
 ```
 If the integrated test is successful, you'll see "Integrated test passed" on the prompt. You can then proceed to the next step. Otherwise, please follow the on-screen instructions to check the log and fix your code / dependencies, and repeat this step until the output is error free.
 
+If there are no errors in the log file apart from workers crashing and the integrated test fails, ensure that you have sufficient resources allocated to docker.
```
Comment (TristanThrush) [inline README.md:57]: I like that you wrote something like this. Consider breaking this up into multiple sentences so it is easier to parse. What do you think of this? Also wondering what @mazhiyi thinks.

"It is possible that your machine could lack the resources to run the deployment environment in docker. If this happens, your log file will show that workers crashed but will not include an error that references your handler.py. Uploading your model could result in fully functional behavior on our server in this scenario, even though the integrated test fails."

File: examples/electra_style_qa/handler.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        Post process inference output into a response.
+        """
+        result = inference_output
+        answer = result['answer']
+        conf = result['score']
+
+        response = dict()
+        example = self._read_data(data)
+        response["id"] = example["uid"]
+        response["answer"] = answer if answer != '[CLS]' else ''
```
Comment (TristanThrush) [inline examples/electra_style_qa/handler.py:57]: Does the pipeline already take care of this CLS case?

**Review by TristanThrush — CHANGES_REQUESTED** [review]
Pretty minimal comments. Otherwise, it looks good!

File: README.md (hunk 1 above)
Comment (mazhiyi) [reply README.md:57]: I think you can add this to the overall `README.md` if you are confident about it. 

As I said we are not able to completely reproduce AWS setup and integrated test is a close proxy. Adding demo dataset as we discussed may be a workaround for this.

File: examples/electra_style_qa/handler.py (hunk 2 above)
Comment (maxbartolo) [reply examples/electra_style_qa/handler.py]: Nice spot, updated!

File: README.md (hunk 1 above)
Comment (maxbartolo) [reply README.md]: Thanks both, updated!

**Review by TristanThrush — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**